controllers/farmer/statistics.js

```diff
@@ -0,0 +1,53 @@
+const { models } = require('../../models');
+const { createError, GENERIC_ERROR } = require('../../helpers/error.js');
+const calculateAge = require('../../helpers/calculateAge');
+
+/**
+ * @description Get farmers statistic
+ *
+ * @param {*} req
+ * @param {*} res
+ * @param {*} next
+ */
+const statistics = async (req, res, next) => {
+  try {
+    const allFarmers = await models.Farmer.find({});
```

Comment:
This needs to filter farmers based on whether they were deleted `archived: false`.

Reply:
Done